openshift-ci/Dockerfile.tools

```diff
@@ -19,7 +19,7 @@ RUN yum --setopt=install_weak_deps=False -y install \
     findutils \
     python2 \
     && yum clean all
-
+#test docker tools
```

Comment:
I'm not sure I get this, could you please elaborate?

Reply:
This is a semi dummy PR to see the current status of 4.7 (just a psudo comment)